src/main/java/ai/lucidtech/las/sdk/Client.java

```diff
@@ -174,6 +187,18 @@ public JSONObject createPrediction(String documentId, String modelName, Map<Stri
         return new JSONObject(jsonResponse);
     }
 
+    /**
+     * Create a prediction on a document specified by path using specified model.
+     * This method takes care of creating and uploading a document specified by document_path.
+     * as well as running inference using model specified by model_name to create prediction on the document.
```

Comment:
Capital letters after punctuation mark. In my opinion 4 occurrences of "specified" in three sentences is too much.  A suggestion for the first sentence:

```suggestion
     * Create a prediction on document <i>documentPath</i> using model <i>modelName</i>.
     ```